roles/confluent.schema-registry/defaults/main.yml

```diff
@@ -7,6 +7,9 @@ schema:
     config_file: /etc/schema-registry/schema-registry.properties
```

Comment:
why doesn't schema_registry_listener_port fit into the variable scheme

Reply:
Do you mean the format of the variable itself?

Reply:
yes

Reply:
well fix it when we go thru all the vars